xbmc/Application.cpp

```diff
@@ -4192,9 +4192,12 @@ void CApplication::ConfigureAndEnableAddons()
 
   if (addonMgr.GetDisabledAddons(disabledAddons) && !disabledAddons.empty())
   {
-    // only look at disabled addons with disabledReason == NONE
-    // usually those are installed from package managers or manually. omit add-ons of type dependecy
-    // also try to enable add-ons with disabledReason == INCOMPATIBLE at startup
+    // this applies to certain platforms only:
+    // look at disabled addons with disabledReason == NONE, usually those are installed via package managers or manually.
+    // also try to enable add-ons with disabledReason == INCOMPATIBLE at startup for all platforms.
+
+    bool isConfigureAddonsAtStartupEnabled =
+        CServiceBroker::GetPlatform().IsConfigureAddonsAtStartupEnabled();
```

Comment:
If you replace this line by just `m_ServiceManager->GetPlatform()` you can even eliminate the `CServiceBroker` changes you introduced with this PR. ;-)